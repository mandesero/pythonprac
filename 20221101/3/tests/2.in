f, g, h = Grange(6, 3, 108575), Grange(7, 5, 104856), Grange(11, 11, 10485577)
o, e = Grange(13, 22, 11), Grange(100, 10, 200)
print(f"{f=}, {g=}, {h=}")
print(f"{list(o)}({not(o)}), {list(e)}({not(e)})")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(h))=}")
print(f"{f[2]=}, {h[33]=}")
a, b = f[3:3276], h[3:3267:5]
print(f"{a=}, {a[1]=}")
print(f"{b=}, {b[2]=}, {list(b)=}")