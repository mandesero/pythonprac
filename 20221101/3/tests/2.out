f=grange(6, 3, 108575), g=grange(7, 5, 104856), h=grange(11, 11, 10485577)
[](True), [100](False)
len(f)=9, len(g)=6, len(h)=6
len(list(f))=9, len(list(g))=6, len(list(h))=6
f[2]=54, h[33]=255476698618765889551019445759400441
a=grange(3, 3, 3276), a[1]=9
b=grange(3, 161051, 3267), b[2]=77812273803, list(b)=[3]
